### PR TITLE
Merge documentation into dev branch

### PR DESCRIPTION
**Pull request by thorstenmaly**

### CODE REVIEW

File: docs/source/auto_examples/plot_exampleDNPOnOff.py.md5 [hunk 1]
```diff
@@ -0,0 +1 @@
+2402ad3b5327595a4170d929660569bc
```
Comment (ylin00) [inline docs/source/auto_examples/plot_exampleDNPOnOff.py.md5]: @thorstenmaly Is this .md5 file needed? You can remove it by 
```
git rm docs/source/auto_examples/plot_exampleDNPOnOff.py.md5
git commit -m 'GIT: rm unused file'
git push
```

Comment (ylin00) [reply docs/source/auto_examples/plot_exampleDNPOnOff.py.md5]: Same with the pickle file below
```docs/source/auto_examples/plot_exampleDNPOnOff_codeobj.pickle```

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (ylin00) [inline .vscode/settings.json]: @thorstenmaly Is this file supposed to be added? Please see comment below to drop it.

**Comment by ylin00** [comment]
@thcasey3 and @tkeller12 , This PR removes a few example scripts. Please check if it works for you.

**Review by tkeller12 — APPROVED** [review]
Everything looks fine.

**Comment by ylin00** [comment]
@thorstenmaly Would you like to delete the `Documentation` branch?

**Pull request merged**